app/grandchallenge/retina_core/management/commands/setannotationpermissions.py

```diff
@@ -0,0 +1,117 @@
+from django.core.management.base import BaseCommand
+
+from config.settings import PERMISSION_TYPES
+from grandchallenge.annotations.models import (
+    MeasurementAnnotation,
+    BooleanClassificationAnnotation,
+    IntegerClassificationAnnotation,
+    PolygonAnnotationSet,
+    LandmarkAnnotationSet,
+    ETDRSGridAnnotation,
+    CoordinateListAnnotation,
+    SinglePolygonAnnotation,
+    SingleLandmarkAnnotation,
+)
+from django.contrib.auth.models import Group
+from django.conf import settings
+from guardian.shortcuts import assign_perm, remove_perm
+
+# Existing annotation (name, codename) as of annotations.0001_initial
+ANNOTATION_CODENAMES = (
```

Comment:
These can be found from the `_meta` class of the objects.

Reply:
Done in a390029ac705996ea10dd8ef173b8bac8bd64e36